### ISSUE TITLE
Improve algorithm of evaluating string similarities

### ISSUE THREAD

**Issue by sleshJdev** [itechartbigdatalab/NY-Accidents-Rate#9]
I see quite straightforward [algorithm](https://github.com/itechartbigdatalab/NY-Accidents-Rate/blob/e3cdc803b9ac928c0882df775fe4324dbaa4a9ac/src/main/scala/com/itechart/ny_accidents/utils/StringUtils.scala#L15) for evaluating score of string similarities. Don't you think about [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance)? It's more robust algorithm. There is great [library](https://github.com/rockymadden/stringmetric) with a lot of handy algorithms:
```
LevenshteinMetric.compare("sitting", "kitten") // 3
LevenshteinMetric.compare("cake", "drake") // 2
```
What do you think?

**Comment by VladimirBaklan** [comment]
Ok, thank you. I will replace it in next PR

**Comment by sleshJdev** [comment]
Thanks a lot. Please refer to this issue in the PR.

**Comment by VladimirBaklan** [comment]
https://github.com/itechartbigdatalab/NY-Accidents-Rate/blob/0fd0ea8d3c85a52bf322c5744aada3f1c2919b78/src/main/scala/com/itechart/ny_accidents/utils/StringUtils.scala#L17

**Comment by sleshJdev** [comment]
Resolved in context of #10
